tilavarauspalvelu/integrations/email/main.py

```diff
@@ -507,7 +506,8 @@ def send_seasonal_reservation_rejected_series_email(
         if reservation.type != ReservationTypeChoice.SEASONAL:
             return
 
-        recipients = get_reservation_email_recipients(reservation=reservation)
+        application = reservation_series.actions.get_application_section().application
```

Comment:
Could just use the ORM here: `reservation_series.allocated_time_slot.reservation_unit_option.application_section.application`.